lib/preloader/actions.js

```diff
@@ -131,6 +131,7 @@ export default function preload({ commit, dispatch }) {
     // add preloaded data to the store
     commit(PRELOAD_SUCCESS, _.assign(tabulaRasa, {
       components: _.assign(components, composeLayoutData(layoutSchema, components, data)),
+      componentVariations: _.get(data, '_componentVariations'),
```

Comment:
you should add `componentVariations` to the default (`tabulaRasa`) file and document them there

Reply:
done